### PR TITLE
Remove trailing space from pyflakes project

### PR DESCRIPTION
**Pull request by srivama**
This redirects the user from link
http://projects.coala.io/#/projects?project=integrate_pyflakes-enhanced_ast_into_coala_
to http://projects.coala.io/#/projects?project=integrate_pyflakes-enhanced_ast_into_coala

Fixes https://github.com/coala/projects/issues/573

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 [build]
   publish = "_site"
   command = "gem install bundler && .ci/build.sh"
+
+[[redirects]]
+  from = "/#/projects?project=integrate_pyflakes-enhanced_ast_into_coala"
```
Comment (jayvdb) [inline netlify.toml]: to & from need to be reversed.

and you need to edit the .md to remove the unwanted space

Comment (srivama) [reply netlify.toml]: The link with _ is the working one, so I redirected the one without _ (not working), to the one with _

**Comment by jayvdb** [comment]
netlify redirect isnt working.

see https://github.com/coala/projects/issues/573#issuecomment-412422531

**Comment by jayvdb** [comment]
Interesting.

I dont seem to be able to get the popup to work with other pages.

e.g. https://deploy-preview-637--coala-projects.netlify.com/#/projects?project=improve_generic_bear_quality

**Comment by jayvdb** [comment]
See it works on master https://coala-projects.netlify.com/#/projects?project=improve_generic_bear_quality

File: resources/js/app.js [hunk 2]
```diff
@@ -262,6 +269,11 @@
 
                     var project_requested = encodeURIComponent($location.search().project);
                     if(project_requested){
+
```
Comment (shikharvaish28) [inline resources/js/app.js]: IMO we don't this extra line here.

File: resources/js/app.js [hunk 3]
```diff
@@ -262,6 +269,11 @@
...
+                        if(project_requested in redirectTo){
+                          project_requested = redirectTo[project_requested]
+                        }
+
```
Comment (shikharvaish28) [inline resources/js/app.js]: (Same here)

**Comment by srivama** [comment]
Yes, it's an old commit. I'll rebase once the code changes are approved. John needs to deploy and verify that the redirects are working properly.

**Comment by shikharvaish28** [comment]
The netlify deploy is working. See the test below.

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/637/commits/1d3d026c19647a9138fda7ba0e841f4ceabc61aa

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**